### PR TITLE
doc/cephfs: simplify ceph fs volume create command

### PR DESCRIPTION
**Pull request by ktdreyer**
Make it easy for new users to choose a sensible default "cephfs" filesystem name.

### CODE REVIEW

**Comment by anthonyeleven** [comment]
IMHO it then becomes unclear that one _can_ specify a different name.  This sort of convention is common in CLI documentation.

**Comment by ktdreyer** [comment]
The purpose of this change is to make it easier for new users. Every choice we present to new users is an additional hurdle that contributes to the feeling that Ceph is hard.

I've updated the PR to add a link to the full [fs-volumes](https://docs.ceph.com/en/latest/cephfs/fs-volumes/) page. This should read as "advanced" information, not something we put on the CephFS index page.

**Comment by anthonyeleven** [comment]
Thanks, that helps.  @zdover23 Do you have any thoughts?

**Comment by dparmar18** [comment]
I think the existing one is still understandable. If we want to make it more easy, this can also be done:

```
ceph fs volume create <fs name>    #where <fs name > can be any name
```

Rationale being: let's say im a complete beginner, if i see `ceph fs volume create cephfs`, I would think okay so `cephfs` is what I need to have. Yeah the inclusion of ceph-volume link might help here, but if there's someone like who just jumps to things, i might skip reading it. Just an opinion here.

**Comment by ktdreyer** [comment]
I jump to things as well. I want to quickly copy and paste. I don't want to paste `<` or `>` characters into my terminal accidentally.

I've updated the PR again to include a comment above the command to explain what it's doing.

**Comment by zdover23** [comment]
> Thanks, that helps. @zdover23 Do you have any thoughts?

My first thought is that I should probably do a better job promulgating the conventions that govern the upstream Ceph documentation.

I also think that I can lay claim to more recently having been a complete beginner than can anyone else in this discussion, and I have precious little sympathy for complete beginners who "jump" to commands and cut and paste them without having understood those commands in context.

**Review by zdover23 — CHANGES_REQUESTED** [review]

**Comment by zdover23** [comment]
> IMHO it then becomes unclear that one _can_ specify a different name. This sort of convention is common in CLI documentation.

I agree with @anthonyeleven that (as I read it, and as I try to put myself in the mind of a first-time reader) the new formulation suggests that the only possible filesystem name is "cephfs".

I suggest that this model be used instead: https://docs.ceph.com/en/quincy/rbd/rados-rbd-cmds/#create-a-block-device-user

In this model, we give the general formula first and then a specific example command second.

(I know that this RBD-related example involves a rather long command with an impressive train of specifications that stretch across the horizon of the website, but it's an example that I have recently touched that seems apropos.)

**Comment by ktdreyer** [comment]
I've updated the PR again to use the sphinx-prompt module and make it clearer that `cephfs` is an example name, not the only possible name.

File: doc/cephfs/index.rst [hunk 1]
```diff
@@ -42,19 +42,21 @@ For most deployments of Ceph, setting up a CephFS file system is as simple as:
 
 .. code:: bash
```
[the remark this answers is not in the document]
Comment (ktdreyer) [reply doc/cephfs/index.rst]: Thanks, I've switched to this one.

**Comment by vshankar** [comment]
@zdover23 If the changes look fine, feel free to merge.

**Comment by anthonyeleven** [comment]
LGTM, thanks!

**Review by anthonyeleven — APPROVED** [review]

**Pull request merged**